01-ruby-fundamentals/exercises/while-worksheet.md

```diff
@@ -0,0 +1,55 @@
+# While Loop Evaluation Worksheet
+For this exercise, you will be evaluating some Ruby code and coming up with scenarios for how the code will execute.
+
+1. For the code below, describe what the loop will do with the following input to the `word` variable. Be sure to include how many times the loop will execute.
+  1. "ham"
+  2. "hamster"
+  3. "bananas"
```

Comment:
Looking at this again, I guess I'm still confused about this one. Are these the words that are provided by the user for one run of the program, or do these represent 3 separate runs? In either case I'm not sure they make sense:
- If it's all the words for one run, then only the first will be used
- If it's 3 separate runs, then what happens in the last run after "bananas" is rejected?